shared/presentation/src/commonMain/kotlin/network/bisq/mobile/presentation/ui/uicases/startup/TrustedNodeSetupPresenter.kt

```diff
@@ -59,10 +59,21 @@ class TrustedNodeSetupPresenter(
             }
 
             settingsRepository.update(updatedSettings)
+
+            showSnackbar("Connected successfully")
+            // showSnackbar("Connected successfully and long text message with long list of english words")
         }
     }
 
     override fun navigateToNextScreen() {
         rootNavigator.navigate(Routes.CreateProfile.name)
     }
+    
+    override fun openQrScanner() {
```

Comment:
please remove all QR code releated code from this PR as agreed

Reply:
Done